## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 474666`
Commit: `9b5cc841e269cc73e75cb836a5fb719b47e6622e`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 3 packages built:</summary>
  <ul>
    <li>age</li>
    <li>age-plugin-tpm</li>
    <li>passage</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 3 packages failed to build:</summary>
  <ul>
    <li>age</li>
    <li>age-plugin-tpm</li>
    <li>passage</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 3 packages built:</summary>
  <ul>
    <li>age</li>
    <li>age-plugin-tpm</li>
    <li>passage</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 3 packages built:</summary>
  <ul>
    <li>age</li>
    <li>age-plugin-tpm</li>
    <li>passage</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>age</summary>
<pre>Building subPackage ./tag
Building subPackage ./tag/internal/age-plugin-tagtest
Building subPackage ./tag/internal/tagtest
Running phase: checkPhase
@nix {&quot;action&quot;:&quot;setPhase&quot;,&quot;phase&quot;:&quot;checkPhase&quot;}
ok  	filippo.io/age	4.652s
ok  	filippo.io/age/agessh	0.315s
ok  	filippo.io/age/armor	0.010s
--- FAIL: TestScript (0.00s)
    --- FAIL: TestScript/scrypt (0.50s)
        testscript.go:584: # encrypt with a provided passphrase (0.070s)
            &gt; stdin input
            &gt; ttyin terminal
            &gt; age -p -o test.age
            &gt; ttyout &#x27;Enter passphrase&#x27;
            FAIL: testdata/scrypt.txt:8: no match for `Enter passphrase` found in ttyout
            
FAIL
FAIL	filippo.io/age/cmd/age	1.062s
FAIL</pre>
</details>
</details>

--- a/pkgs/by-name/ag/age/package.nix
+++ b/pkgs/by-name/ag/age/package.nix
@@ -18,21 +18,21 @@
 
 buildGoModule (final: {
   pname = "age";
-  version = "1.2.1";
+  version = "1.3.1";
 
   src = fetchFromGitHub {
     owner = "FiloSottile";
     repo = "age";
     tag = "v${final.version}";
-    hash = "sha256-9ZJdrmqBj43zSvStt0r25wjSfnvitdx3GYtM3urHcaA=";
+    hash = "sha256-Qs/q3zQYV0PukABBPf/aU5V1oOhw95NG6K301VYJk8A=";
   };
 
-  vendorHash = "sha256-ilRLEV7qOBZbqzg2XQi4kt0JAb/1ftT4JmahYT0zSRU=";
+  vendorHash = "sha256-iVDkYXXR2pXlUVywPgVRNMORxOOEhAmzpSM0xqSQMSQ=";
 
   ldflags = [
     "-s"
     "-w"
-    "-X main.Version=${final.version}"
+    "-X main.Version=v${final.version}"
   ];
 
   nativeBuildInputs = [ installShellFiles ];

